sdk/core/core-amqp/src/errors.ts

```diff
@@ -435,38 +435,94 @@ export enum ErrorNameConditionMapper {
   SystemError = "system:error"
 }
 
+/**
+ * Describes the fields on a Node.js SystemError.
+ * Omits fields that are not related to network calls (e.g. file system calls).
+ * See https://nodejs.org/dist/latest-v12.x/docs/api/errors.html#errors_class_systemerror
+ */
+interface NetworkSystemError {
+  address?: string;
+  code: string;
+  errno: string | number;
+  info?: any;
+  message: string;
+  name: string;
+  port?: number;
+  stack: string;
+  syscall: string;
+}
+
+const systemErrorFieldsToCopy: (keyof Omit<NetworkSystemError, "name" | "message">)[] = [
+  "address",
+  "code",
+  "errno",
+  "info",
+  "port",
+  "stack",
+  "syscall"
+];
+
 /**
  * Describes the base class for Messaging Error.
  * @class {MessagingError}
  * @extends Error
  */
 export class MessagingError extends Error {
   /**
-   * @property {string} [condition] The error condition.
+   * Address to which the network connection failed.
+   * Only available in Node.js.
    */
-  condition?: string;
+  address?: string;
+  /**
+   * A string label that identifies the error.
+   */
+  code?: string;
+  /**
+   * System-provided error number.
+   * Only available in Node.js.
+   */
+  errno?: number | string;
   /**
    * @property {string} name The error name. Default value: "MessagingError".
    */
   name: string = "MessagingError";
   /**
-   * @property {boolean} translated Has the error been translated. Default: true.
+   * The unavailable network connection port.
+   * Only available in Node.js.
+   */
+  port?: number;
+  /**
+   * Name of the system call that triggered the error.
+   * Only available in Node.js.
    */
-  translated: boolean = true;
+  syscall?: string;
   /**
    *
    * @property {boolean} retryable Describes whether the error is retryable. Default: true.
    */
   retryable: boolean = true;
   /**
-   * @property {any} [info] Any additional error information given by the service.
+   * @property {any} [info] Extra details about the error.
    */
   info?: any;
   /**
    * @param {string} message The error message that provides more information about the error.
+   * @param originalError An error whose properties will be copied to the MessagingError if the
```

Comment:
This statement is no longer true as we are now copying only `systemErrorFieldsToCopy`

Reply:
Updated to be more accurate.